<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Ahmeds-PC.local" failures="1" tests="2" name="End to End Test " time="59.68" errors="0" timestamp="2022-10-09T03:09:43 EET">
  <testcase classname="com.AmazonApp.testcases.LoginPageTest" name="LoginTest" time="3.54"/>
  <testcase classname="com.AmazonApp.testcases.TodayDealsTest" name="TodayDeals" time="22.125">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [true] but found [false]">
        <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [true] but found [false]
at com.AmazonApp.testcases.TodayDealsTest.TodayDeals(TodayDealsTest.java:139)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- TodayDeals -->
</testsuite> <!-- End to End Test  -->
